[JENKINS-32701] Escape percentage signs in the jenkins-wrapper batch

arthurschreiber opened the pull request:
When writing workspace paths into the `jenkins-wrapper.bat` batch
script, percentage signs need to be escaped correctly. Otherwise
the batch script execution will silently fail when the workspace
path contains a percentage sign. This was encountered in a multi branch
workflow project.

arthurschreiber commented:
Hey @jglick - is anything keeping this from being merged?

jglick commented:
A merge conflict with `master`, and also there is no test reproducing the problem and confirming the fix.

arthurschreiber commented:
Hey @jglick, I've rebased this against the latest master changes and added a test case that will hang forever without this fix in place. Could you please merge this and push out a release ASAP? :heart:

arthurschreiber commented:
@jglick Ping?

jglick commented:
Linking to existing issue [JENKINS-32701](https://issues.jenkins-ci.org/browse/JENKINS-32701).

jglick commented:
Seems to work on both XP and 2012.

jglick commented:
Released as 1.9. Thanks, and sorry for the delay (I have quite a backlog).
